gradlew

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 #
-# Copyright � 2015-2021 the original authors.
+# Copyright © 2015-2021 the original authors.
```

Comment:
Weird, the diff should not exist, but there is no evident diffs, so it should be ok

Reply:
True, it's weird